app/views/draws/show.html.erb

```diff
@@ -19,7 +19,10 @@
   <% end %>
 </div>
 <% if policy(@draw).activate? %>
-  <%= link_to 'Begin draw process', activate_draw_path(@draw), method: :patch %>
+  <%= link_to 'Begin draw process<br>'.html_safe, activate_draw_path(@draw), method: :patch %>
+<% end %>
+<% if policy(@draw).update? %>
+  <%= link_to 'Add group to draw<br>'.html_safe, new_draw_group_path(@draw) %>
```

Comment:
Why are we adding line breaks to the link text? We should either do it outside or simply style these links with CSS to have some more padding.